src/Compute/v2/Models/QuotaSet.php

```diff
@@ -0,0 +1,170 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Compute\v2\Models;
+
+use OpenStack\Common\Resource\Deletable;
+use OpenStack\Common\Resource\OperatorResource;
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\Updateable;
+
+/**
+ * Represents a Compute v2 Quota Set
+ *
+ * @property \OpenStack\Compute\v2\Api $api
+ */
+class QuotaSet extends OperatorResource implements Retrievable, Updateable, Deletable
+{
+    /**
+     * The number of allowed instance cores for each tenant.
+     *
+     * @var  int|array
+     */
+    public $cores;
+
+    /**
+     * The number of allowed fixed IP addresses for each tenant.
+     * Must be equal to or greater than the number of allowed instances.
+     *
+     * @deprecated Since 2.35
+     * @var  int|object
+     */
+    public $fixedIps;
+
+    /**
+     * The number of allowed floating IP addresses for each tenant.
+     *
+     * @deprecated Since 2.35
```

Comment:
See also https://developer.openstack.org/api-ref/compute/?expanded=show-the-detail-of-quota-detail#show-the-detail-of-quota

I think the logic here is that network related quotas such as `floating_ips` and `fixed_ips` shall be managed by Networking service;

I put `@deprecated` flag here to discourage people in using these attributes.